Dockerfile

```diff
@@ -19,7 +19,6 @@ RUN chmod +x /metrics/source/app/action/index.mjs \
   # Based on https://github.com/github/linguist and https://github.com/github/licensed
```

Comment:
Whichever the Linguist imports are can be removed. Line 21?

Reply:
I don't remember which imports is for which gem unfortunately, but most of these dependencies are probably already on the base image or used somewhere else so I don't think it's too problematic to let them

Reply:
Linguist wants `sudo apt-get install cmake pkg-config libicu-dev zlib1g-dev libcurl4-openssl-dev libssl-dev ruby-dev` while Licensed just wants `sudo apt-get install cmake pkg-config`. I think everything from libicu-dev onwards can be removed.

Reply:
Yes we should remove it so 👍

Reply:
Opened #458 :+1: